tests/plugins/font/font.js

```diff
@@ -170,6 +172,43 @@
 				'<p>x<span style="font-size:12px"><em>foo</em></span><em><span style="font-size:24px">bar</span></em>x@</p>' );
 		},
 
+		// #584
```

Comment:
Is there any test checking if second click (reapplying) of the same font/size doesn't change the content?